croud/projects/users/commands.py

```diff
@@ -19,48 +19,31 @@
 
 from argparse import Namespace
 
-from croud.gql import Query, print_query
+from croud.rest import Client
+from croud.session import RequestMethod
 
 
 def project_user_add(args: Namespace) -> None:
     """
     Adds a user to a project.
     """
 
-    _query = f"""
-    mutation {{
-        addUserToProject(input: {{
-            projectId: "{args.project_id}",
-            user: "{args.user}"
-        }}) {{
-            success
-        }}
-    }}
-    """
-
-    query = Query(_query, args)
-    query.execute()
-    print_query(query, "addUserToProject", "Successfully added user to project.")
+    client = Client(env=args.env, output_fmt=args.output_fmt)
+    client.send(
+        RequestMethod.POST,
+        f"/api/v2/projects/{args.project_id}/users/",
+        body={"user": args.user},
+    )
+    client.print(keys=["user", "role_fqn"])
```

Comment:
Contrary to the organization's user adding, this doesn't return the project ID. Let's change that.

Reply:
So you want to update the backend to return the `project_id` in addition?

Reply:
Err, sorry. I meant "print", not "return".
```suggestion
    client.print(keys=["project_id", "user", "role_fqn"])
```